src/Rendering/prePassRendererSceneComponent.ts

```diff
@@ -0,0 +1,201 @@
+import { Nullable } from "../types";
+import { Scene } from "../scene";
+import { ISceneSerializableComponent, SceneComponentConstants } from "../sceneComponent";
+import { PrePassRenderer } from "./prePassRenderer";
+import { AbstractScene } from "../abstractScene";
+import { Color3 } from "../Maths/math.color";
+import { Logger } from "../Misc/logger";
+
+declare module "../abstractScene" {
+    export interface AbstractScene {
+        /** @hidden (Backing field) */
+        _prePassRenderer: Nullable<PrePassRenderer>;
+
+        /**
+         * Gets or Sets the current prepass renderer associated to the scene.
+         */
+        prePassRenderer: Nullable<PrePassRenderer>;
+
+        /**
+         * Enables the prepass and associates it with the scene
+         * @returns the PrePassRenderer
+         */
+        enablePrePassRenderer(): Nullable<PrePassRenderer>;
+
+        /**
+         * Disables the prepass associated with the scene
+         */
+        disablePrePassRenderer(): void;
+
+        /**
+         * Diffusion profile colors for subsurface scattering
+         * You can add one diffusion color using `addDiffusionProfile` on `scene.prePassRenderer`
+         * See ...
+         * Note that you can only store up to 5 of them
+         */
+        ssDiffusionProfileColors: Color3[];

```

Comment:
Shouldnt this be part of the subsurfaceConfiguration in the PrePassRenderer ?

Reply:
I put it in the scene in order for that variable to be serialized. It reuses the pattern from `AudioSceneComponent`.
Maybe we should look into how to serialize infos in the component itself ? I don't think it has been done yet

Reply:
nevermind i found a way !